.github/workflows/release.yml

```diff
@@ -94,13 +95,6 @@ jobs:
           --title 'PUDL ${{ github.ref_name }}'
           --generate-notes
           --repo '${{ github.repository }}'
-      - name: Tag the current stable commit
-        run: |
-          git config user.email "pudl@catalyst.coop"
-          git config user.name "pudlbot"
-          git checkout stable
-          git merge --ff-only ${{ github.ref_name }}
-          git push
```

Comment:
All this logic now in the build script, where it can be conditional on the ETL actually succeeding.